platooning_control_ihp/package.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<package format="3">
+  <name>platoon_control_ihp</name>
+  <version>3.3.0</version>
```

Comment:
A new package should start over versioning from 1.0.0, yes?
Is there something in the C++ code that also tells its version to health monitor?  It's problematic to have version ID source in 2 places, because they can get out of sync.  This is a problem beyond the scope of this PR, but may need some thought.

Reply:
changed to 1.0.0
I dont think that version is used anywhere, it is used only for record keeping